spec/system/admin/reports_spec.rb

```diff
@@ -31,6 +31,21 @@
     end
   end
 
+  describe "Background processing" do
+    before do
+      Flipper.enable(:background_reports)
+      ActiveJob::Base.queue_adapter.perform_enqueued_jobs = true
+    end
+
+    it "can run the customers report" do
+      login_as_admin_and_visit admin_report_path(
+        report_type: :customers, report_subtype: :mailing_list
+      )
+      click_button "Go"
+      expect(page).to have_content "EMAIL FIRST NAME"
+    end
+  end
+
```

Comment:
Because we're passing report parameters and chosen format to the job, it might be worth testing that these options have been used.
But I wouldn't suggest that prevents us from merging this, this is just a thought maybe for the next iteration.